src/Common/ExtensionMethods.cs

```diff
@@ -84,7 +84,7 @@ public static string RemoveInvalidPathChars(this string input)
         /// <returns><code>input</code> without invalid path characters.</returns>
         public static string FixJobName(this string input)
         {
-            var jobName = ValidJobNameRegex.Replace(input, "-").TrimStart('-');
+            var jobName = ValidJobNameCharactersRegex.Replace(input, "-").TrimStart('-');
```

Comment:
The method summary looks to be copy-pasted here, and is not correct. It would also be better to explain valid chars are dictated/limited by the dependency API.